Remove composer.lock from repo

cmodijk opened the pull request:
Remove composer.lock from repo and i modified the .gitignore to exclude it

matthiasnoback commented:
Could you also do a `git rm --force composer.lock`? Otherwise the file would still exist in the repo.

cmodijk commented:
I already did the I removed composer.lock file and updated the .gitignore if i try to use `git rm --force composer.lock` i get `fatal: pathspec 'composer.lock' did not match any files`

matthiasnoback commented:
Oh, oops, sorry, didn't see that you already had 2 changed files... Will merge right-away.
